man/io_uring_setup.2

```diff
@@ -148,6 +149,27 @@ The
 .I resv
 array must be initialized to zero.
 
+.I features
+is filled in by the kernel, which specifies various features supported
+by current kernel version.
+.TP
+.B IORING_FEAT_SINGLE_MMAP
+.TP
```

Comment:
You should add here:

If this flag is set, the two SQ and CQ rings can be mapped with a single
.I mmap(2)
call. The SQES must still be allocated separately. This brings the neccessary
.I mmap(2)
calls down from three to two.